ci-operator/config/openshift/platform-operators/openshift-platform-operators-main.yaml

```diff
@@ -11,7 +11,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.18
```

Comment:
Can you use this instead?
```
build_root:
  from_repository: true
```

Reply:
Presumably, this will keep up with the value in go.mod, and thus, never need to be modified again?

Reply:
Close; it uses https://github.com/openshift/platform-operators/blob/main/.ci-operator.yaml

Reply:
From whatever branch is appropriate

Reply:
Better than having to modify this repo...

Reply:
I can't. Go 1.20 from the repo fails CI. And 1.18 also fails CI.

I have another PR open to switch to from_repository.

Reply:
Gotcha.

Reply:
Might need more surgery though. Maybe need to temporarily disable the lint test. Or force a go bump.

Reply:
I figured out at least one problem. platform-operators tries to use `setup-envtest@latest`, which apparently now requires a more recent version of Go to work.

I've updated @awgreene's PR (openshift/platform-operators#92) to pin setup-envtest. And I've also opened a separate PR to switch CI tooling over to bingo (openshift/platform-operators#95) to make sure build tools actually stay pinned.

We _may_ still need this PR because there is also an interplay between golangci-lint and the go compiler version that may require an intermediate bump to Go 1.19.